components/test_backup/Cargo.toml

```diff
@@ -30,3 +30,5 @@ tidb_query_common = { path = "../tidb_query_common" }
 tikv = { path = "../../", default-features = false }
 tikv_util = { path = "../tikv_util", default-features = false }
 txn_types = { path = "../txn_types", default-features = false }
+protobuf = "2"
```

Comment:
Where it is used ?

Reply:
i don't remember why it's added, maybe depend on something during coding. Anyway, it's not used for now. I will remove it.

Reply:
protobuf::RepeatedField::from_vec used to construct the test case input in RawChecksumRequest.